pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -426,13 +433,22 @@ func (f *framework) RunPostBindPlugins(
 // returns the error. In such case, pod will not be scheduled.
 func (f *framework) RunReservePlugins(
 	state *CycleState, pod *v1.Pod, nodeName string) *Status {
+	reserved := []string{}
+	defer func() {
+		for _, name := range reserved {
+			if pl, ok := f.unreservePlugins[name]; ok {
```

Comment:
It assumes that the reserved plugin also implements `Unreserve` plugin interface.  It is reasonable, but we do not make a promise about it now. How about adding a restriction that a `Reserve` plugin also need implements `Unreserve` plugin interface (in another PR)? cc @ahg-g @Huang-Wei  

If so, we do not need a map `f.unreservedPlugins` to record it.

Reply:
> How about adding a restriction that a Reserve plugin also need implements Unreserve plugin interface (in another PR)?

Sounds good! It seems to be natural to make `Reserve` and `Unreserve` be pair.